apps/web-connect/src/features/hooks/contract/useMintBatch.ts

```diff
@@ -74,16 +77,19 @@ export function useMintBatch({
   };
 
   const mintBatch = async (qtyToMint: number) => {
+    const totalEthPriceInWei = calculateTotalPrice();
+    const expectedAvg = totalEthPriceInWei / BigInt(qtyToMint);
+    console.log("expectedAvg: ", expectedAvg);
```

Comment:
This is only present to prevent build error for unused variables(while testing). Will be removed prior to PR being ready for review.